src/templates/detail/main.html

```diff
@@ -246,11 +245,58 @@
 {% except %}
 {% end %}
 
-{% include "detail/content_ratings.html" %}
-
-<section class="main full">
-  <a class="button support" href="{{ url('app/abuse', [slug]) }}">
-    {{ _('Report Abuse') }}</a>
+<!--More information (detail/more-info.styl).-->
+<section class="main full app-more-info">
+  <div>
+    {% defer (url=endpoint, as='app', key=slug) %}
+      <div class="detail-flex-wrap">
+        <div class="detail-left-side">
+          <h3>{{ _('More Information') }}</h3>
+          <dl class="app-more-metadata">
+            <div>
```

Comment:
I need to group the elements for styling purposes. I might drop the ```dl``` and go with just ```div```s. It'd be nice to use ```dl```s but they don't support grouping content well.